### ISSUE TITLE
Checkboxes lists : Tabs are no longer visible in the "view mode"

### ISSUE THREAD

**Issue by Jerome-Herbinet** [nextcloud/deck#4012]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

**Describe the bug**
Tabs are no longer visible in display mode, for checkbox lists only (not for bullet lists)

**To Reproduce**
1. Go to an existing or new card
2. In description, create a list of checkboxes by offsetting some of them with tabs
3. Then compare, edit mode and view mode. You will notice that tabs are no longer visible in view mode and that all checkboxes are left aligned, as if no tabs had been used.

Note: If you make a bulleted list, you will see that the tabs appear in view mode in this case. Checkbox lists should take tabs into account in the same way that bulleted lists do. According to a recent testimonial I had, checkbox lists used to take tabs into account in view mode.

**Expected behavior**
Tabs used in checkbox lists should be visible in view mode.

**Screenshots**
Edit mode : 
![edit](https://user-images.githubusercontent.com/33763786/187434581-72a82d9c-20d3-4799-add3-d784fb1a3841.png)

View mode : 
![view](https://user-images.githubusercontent.com/33763786/187434625-dd7f0ef2-0ea3-4829-9402-8455e7953dfa.png)

**Client details:**
 - OS: Ubuntu 20.04 LTS
 - Browser : Firefox 104.0
 - Device: Laptop

<details>
<summary>Server details</summary>

**Operating system**:

**Operating system**: Ubuntu 20.04 LTS

**Web server:** Apache (fpm-fcgi)

**Database:** mysql 10.3.34

**PHP version:** 8.0.22

**Nextcloud version:** 24.0.4.1

**Where did you install Nextcloud from:** Web installer

**Signing status:**

```
No errors have been found.
```

**List of activated apps:**

```
Enabled:
 - accessibility: 1.10.0
 - activity: 2.16.0
 - announcementcenter: 6.3.1
 - apporder: 0.15.0
 - bruteforcesettings: 2.4.0
 - calendar: 3.4.3
 - circles: 24.0.1
 - cloud_federation_api: 1.7.0
 - comments: 1.14.0
 - contacts: 4.2.0
 - contactsinteraction: 1.5.0
 - cookbook: 0.9.13
 - dashboard: 7.4.0
 - dav: 1.22.0
 - deck: 1.7.1
 - external: 4.0.0
 - federatedfilesharing: 1.14.0
 - federation: 1.14.0
 - files: 1.19.0
 - files_accesscontrol: 1.14.1
 - files_automatedtagging: 1.14.0
 - files_external: 1.16.1
 - files_lock: 24.0.1
 - files_pdfviewer: 2.5.0
 - files_readmemd: 1.2.2
 - files_rightclick: 1.3.0
 - files_sharing: 1.16.2
 - files_trashbin: 1.14.0
 - files_versions: 1.17.0
 - files_videoplayer: 1.13.0
 - firstrunwizard: 2.13.0
 - flow_notifications: 1.4.0
 - groupfolders: 12.0.1
 - impersonate: 1.11.0
 - jitsi: 0.15.0
 - logreader: 2.9.0
 - lookup_server_connector: 1.12.0
 - metadata: 0.16.0
 - nextcloud_announcements: 1.13.0
 - notes: 4.5.0
 - notifications: 2.12.0
 - oauth2: 1.12.0
 - occweb: 0.1.0
 - password_policy: 1.14.0
 - passwords: 2022.8.20
 - phonetrack: 0.7.0
 - photos: 1.6.0
 - privacy: 1.8.0
 - provisioning_api: 1.14.0
 - quota_warning: 1.14.0
 - recommendations: 1.3.0
 - richdocuments: 6.2.0
 - richdocumentscode: 22.5.502
 - serverinfo: 1.14.0
 - settings: 1.6.0
 - sharebymail: 1.14.0
 - spreed: 14.0.4
 - support: 1.7.0
 - survey_client: 1.12.0
 - systemtags: 1.14.0
 - text: 3.5.1
 - theming: 1.15.0
 - theming_customcss: 1.11.0
 - twofactor_backupcodes: 1.13.0
 - twofactor_email: 2.5.0
 - twofactor_nextcloud_notification: 3.4.0
 - twofactor_totp: 6.4.0
 - updatenotification: 1.14.0
 - user_status: 1.4.0
 - video_converter: 1.0.5
 - viewer: 1.8.0
 - weather_status: 1.4.0
 - welcome: 1.0.1
 - workflowengine: 2.6.0
Disabled:
 - admin_audit
 - duplicatefinder: 0.0.15
 - encryption
 - extract: 1.3.5
 - files_zip: 1.1.2
 - user_ldap
```

**Nextcloud configuration:**

```
{
    "instanceid": "***REMOVED SENSITIVE VALUE***",
    "passwordsalt": "***REMOVED SENSITIVE VALUE***",
    "secret": "***REMOVED SENSITIVE VALUE***",
    "trusted_domains": [
        "nextcloud.mydomain.fr"
    ],
    "datadirectory": "***REMOVED SENSITIVE VALUE***",
    "dbtype": "mysql",
    "version": "24.0.4.1",
    "updatechecker": true,
    "overwrite.cli.url": "https:\/\/nextcloud.mydomain.fr",
    "dbname": "***REMOVED SENSITIVE VALUE***",
    "dbhost": "***REMOVED SENSITIVE VALUE***",
    "dbport": "",
    "dbtableprefix": "oc_",
    "mysql.utf8mb4": true,
    "dbuser": "***REMOVED SENSITIVE VALUE***",
    "dbpassword": "***REMOVED SENSITIVE VALUE***",
    "installed": true,
    "default_language": "fr",
    "default_locale": "fr_FR",
    "knowledgebaseenabled": false,
    "default_phone_region": "ISO3166-2",
    "mail_from_address": "***REMOVED SENSITIVE VALUE***",
    "mail_smtpmode": "smtp",
    "mail_sendmailmode": "smtp",
    "mail_domain": "***REMOVED SENSITIVE VALUE***",
    "mail_smtpauthtype": "LOGIN",
    "mail_smtpauth": 1,
    "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
    "mail_smtpport": "25",
    "mail_smtpname": "***REMOVED SENSITIVE VALUE***",
    "mail_smtppassword": "***REMOVED SENSITIVE VALUE***",
    "memcache.local": "\\OC\\Memcache\\Redis",
    "memcache.locking": "\\OC\\Memcache\\Redis",
    "redis": {
        "host": "***REMOVED SENSITIVE VALUE***",
        "port": 6379
    },
    "maintenance": false,
    "theme": "",
    "loglevel": 2,
    "updater.release.channel": "stable",
    "app_install_overwrite": [
        "occweb"
    ]
}
```

**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...

No

</details>

<details>
<summary>Logs</summary>

#### Nextcloud log (data/nextcloud.log)
```
Can be provided if necessary
```

#### Browser log
```
Insert your browser log here, this could for example include:

Can be provided if necessary
```

</details>

**Comment by q-wertz** [comment]
Maybe due to https://github.com/nextcloud/deck/pull/3833?
Also not sure how this https://github.com/nextcloud/deck/pull/3898 which is not yet released impacts the behavior.

**Comment by Jerome-Herbinet** [comment]
> Having the same issue. (Also not working with whitespaces.)
> 
> #3833 should have fixed this I think? Also not sure how #3898 (which is not yet released as far as is see) impacts the behavior.

Yes, #3637 PR should fix #3637 issue which correspond to my described problem.
